spec.html

```diff
@@ -41743,7 +41743,7 @@ <h1>_TypedArray_.prototype.BYTES_PER_ELEMENT</h1>
 
       <emu-clause id="sec-typedarray.prototype.constructor">
         <h1>_TypedArray_.prototype.constructor</h1>
-        <p>The initial value of a _TypedArray_`.prototype.constructor` is the corresponding %TypedArray% intrinsic object.</p>
+        <p>The initial value of the *"constructor"* property of the prototype for a given _TypedArray_ constructor is the intrinsic object, associated with the current realm, corresponding to _TypedArray_ (<emu-xref href="#table-well-known-intrinsic-objects"></emu-xref>).</p>
```

Comment:
I'm not very familiar with spec conventions, but I personally read "the prototype for X" as meaning "X.[[Prototype]]" instead of "X`.prototype`".

Reply:
We talked about this in the editor call and we think this is sufficiently unambiguous given the context.